cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1341,6 +1341,32 @@
     </description>
   </property>
 
+  <!-- Operational Stats Configuration -->
+  <property>
+    <name>operational.stats.extensions.dir</name>
+    <value>/opt/cdap/master/ext/operations</value>
+    <description>
+      Semicolon-separated list of local directories that are scanned for
+      operational stats extensions
+    </description>
+  </property>
+
+  <property>
+    <name>operational.stats.collection.threads</name>
```

Comment:
Although I didn't implement that, my initial thought was to collect a single operational stat per thread. I didn't implement that because I felt it would be unnecessary to start with. Forgot to remove this. Removed now. Will think about multiple threads if there really is a use-case for it.